### ISSUE TITLE
No temperature values in 0.4.16

### ISSUE THREAD

**Issue by therealhalifax** [tschamm/boschshc-hass#36]
Since the update to 0.4.16 the temperature of the valves and also the thermostate is not shown anymore.

Sensor not available is shown.

After downgrade to 0.4.12 it works again.

**Comment by NilsGG** [comment]
I had the same error on my System with the temperatures not showing up correctly after the Update to 0.4.16+. It seems to result from the API change in the HA core. @tschamm I have not yet looked at the API change of HA fully, but it seems to be the result of the property name change of [state](https://github.com/tschamm/boschshc-hass/blob/master/custom_components/bosch_shc/sensor.py#L158). Hope it helps.

**Comment by tschamm** [comment]
Thanks for reporting. The defined entity attributes were overwritten by the implementation in the parent class. Fixed by https://github.com/tschamm/boschshc-hass/commit/30d169251fd8eab77687eef96057d6f9b630cf7f

**Comment by tschamm** [comment]
You have to delete the wrongly created entities by hand. Sorry for the trouble!

**Comment by JanZimmer** [comment]
@tschamm I just updated again from 0.4.12 to 0.4.18.

I understand, that `current_temeprature` is now an attribute of `climate.XXX`. However, before this update, there were also `_sensors_` for the temperature called `sensor.XXX_temperature`. These do not get any values after all. 

This is especially bad, as `climate` represents one "room" whereas these sensors were bound to specific thermostats where a room can have multiple of them with different temperature readings.

Am I doing anything wrong here? Do I have to update hass to a specific version?

There are no error in the log. I also attached the Overview of the actual HASS device of one of the thermostats.

![image](https://user-images.githubusercontent.com/12293998/140911685-dc0ff43c-8a06-46f0-b5d7-05b33c37714d.png)
